gtep/gtep_model.py

```diff
@@ -1297,7 +1210,7 @@ def CP_flow_balance(b):
             balance = 0
             # load = m.loads.get(bus) or 0
             buses = [b for b in m.buses]
-            loads = [l for l in m.loads]
+            loads = [l for l in b.loads]
```

Comment:
I think this is a bug. You're overwriting `b` on the previous line when iterating over `m.buses`